src/renderers/shared/fiber/ReactChildFiber.js

```diff
@@ -73,7 +73,20 @@ const {
   Deletion,
 } = ReactTypeOfSideEffect;
 
-function coerceRef(current: Fiber | null, element: ReactElement) {
+function getElementTypeForWarning(element) {
+  if (element === null) {
+    return 'null';
+  }
+  if (element === undefined) {
+    return 'undefined';
```

Comment:
Can we avoid strings in these two cases? Seems like `return '' + element` would be enough. Minor thing but minifies better.